src/components/biggive-carousel/biggive-carousel.tsx

```diff
@@ -0,0 +1,103 @@
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
+
+@Component({
+  tag: 'biggive-carousel',
+  styleUrl: 'biggive-carousel.scss',
+  shadow: true,
+})
+export class BiggiveCarousel {
+  @Element() host: HTMLBiggiveCarouselElement;
+
+  @Prop() spaceBelow: spacingOption = 4;
+
+  @Prop() columnCount: number = 3;
+
+  @Prop() buttonBackgroundColour: brandColour = 'white';
+
+  @Prop() buttonIconColour: brandColour = 'primary';
+
+  currentTab = 0;
+  itemCount = 0;
+  itemWidth = 0;
+  max = 0;
+  min = 0;
```

Comment:
I didn't really follow what max and min are here. Maybe a docblock would help, or maybe they aren't needed in this form? The only assignments I can see to `max` are `this.max = 0` - so why have it as mutable variable if its only ever zero?

Reply:
Done